stdlib/3/os/__init__.pyi

```diff
@@ -359,11 +359,9 @@ def symlink(source: _PathType, link_name: _PathType,
             target_is_directory: bool = ...) -> None:
     ...  # final argument in Windows only
 def unlink(path: _PathType) -> None: ...
-# TODO: add ns, dir_fd, follow_symlinks argument
-if sys.version_info >= (3, 0):
-    def utime(path: _PathType, times: Optional[Tuple[float, float]] = ...) -> None: ...
-else:
-    def utime(path: _PathType, times: Optional[Tuple[float, float]]) -> None: ...
+def utime(path: _PathType, times: Optional[Union[Tuple[int, int], Tuple[float, float]]] = ...,
```

Comment:
Isn't this redundant because `int` is implicitly promoted to `float`? Were you getting a mypy error that you're trying to solve?

Reply:
I don't know what you mean when you say that `int` is implicitly promoted to `float`, but if you pass an `int` to utime, it will be treated differently than if you pass a `float`. The `int` wil keep it's "exactness" and be used to set the utime precisely at x sec and 0 nanosecs. The float will be converted to x secs and x nanosecs.

The reason for the change was that mypy told me to use a float, and that goes against my intuition. Keeping the utime unchanged is important for backup systems and such, and since floats are inherently not perfectly accurate this feels wrong. Even though the float precision is probably enough to keep the nanosecs unchanged.

Reply:
`int` is implicitly promoted to `float` in PEP 484's type system: every function that takes a `float` implicitly also takes an `int`. But there's a reasonable argument to be made that here it's better to be explicit and also mention `int`.